addons/mail/static/src/core/common/discuss_header.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="mail.DiscussHeader">
+        <t t-set="partitionedActions" t-value="props.threadActions.partition"/>
+        <div class="o-mail-Discuss-header px-3 d-flex flex-shrink-0 align-items-center border-bottom z-1 flex-grow-0" t-ref="header">
```

Comment:
Please follow BEM convention in Discuss code:

- Template is named moduleName.ComponentName
- Root node is named o-moduleName-ComponentName
- Parts are named o-moduleName-ComponentName-partName